app/models/manageiq/providers/openstack/inventory_collection_default/cloud_manager.rb

```diff
@@ -217,7 +217,7 @@ def orchestration_stacks_resources(extra_attributes = {})
 
     def orchestration_stacks(extra_attributes = {})
       attributes = {
-        :model_class                 => ManageIQ::Providers::Openstack::CloudManager::OrchestrationStack,
+        :model_class                 => ManageIQ::Providers::CloudManager::OrchestrationStack,
```

Comment:
I see. I was hoping I could use the parent class so that I didn't have to have a separate entry for vnfs. ;)